runtime/lua/vim/treesitter/language.lua

```diff
@@ -2,19 +2,66 @@ local a = vim.api
 
 local M = {}
 
+---@type table<string,string>
+local ft_to_lang = {}
+
+---@param filetype string
+---@return string|nil
+function M.get_lang(filetype)
+  return ft_to_lang[filetype]
+end
+
+---@deprecated
+function M.require_language(lang, path, silent, symbol_name)
```

Comment:
deprecate() ?

Reply:
Our doc flow seems to only recognize `@deprecated`

Reply:
Oh, you mean use `vim.deprecated()`

Reply:
I added a call to `vim.deprecate` but I think the notify is a bit been obtrusive. We can add it later when at least nvim-treesitter adapts. `require_language` is just an alias for `add` anyway.